app/models/journal.rb

```diff
@@ -0,0 +1,3 @@
+class Journal < ApplicationRecord
+  belongs_to :user
```

Comment:
OK, makes sense. I will also add `belongs_to :journalable, polymorphic: true` in `Journal` and `has_many :journals` to `User`, so both relations are defined on a model.